src/lib/mock-backend.js

```diff
@@ -1134,6 +1134,15 @@ var owner = {
     new User('Kadın', 'Kübra Gizem', 'https://randomuser.me/api/portraits/women/84.jpg')]
 };
 
+var ScreenTypes = {
+        UPTODATE: 1,
+        ARCHIEVE: 2,
+        RECYCLEBIN: 3,
+        USER: 4,
+        PROFILE: 5,
+        SEARCH: 6
+    }
+
```

Comment:
`        this.EventType = {
            SET_ACTIVE_THREAD: 'set active thread',
            UPDATE: 'update',
            SET_ACTIVE_CHAT_BOX: 'set active chat box',
            NEW_MESSAGE: 'new message',
            ADD_CHAT_BOX: 'add chat box',
            REMOVE_CHAT_BOX: 'remove chat box'
        };`
@ezgicatalbas we can define ENUM in service card or wherever you want. Teacher has already share with us like above code. This will be more meaningful instead using array.

you can examine it in week-3-2 in card-service.js